source/monsters/index.rst

```diff
@@ -20,26 +20,16 @@ Monsters
    legendary-creatures
    statistics
    
-.. toctree::
-    :glob:
-    :maxdepth: 1
-
-    Monster Descriptions <monsters_a-z/index>
-
-------------------------------------------
-
-Monster Descriptions
-=====================
+.. container:: h1
 
+   Monster Descriptions
 
-.. container:: columns
-
+.. container:: subcolumns  
+   
     .. toctree::
-      :name: monsterlist
-      :titlesonly:
-      :glob:
-      :maxdepth: 3
-
-      monsters_a-z/*/index
-      Miscellaneous Creatures <miscellaneous-creatures/index>
-      Nonplayer Characters <nonplayer-characters/index>
+        :name: monster-descs
+        :glob:
+        :titlesonly:
+        :maxdepth: 3
+
+        Monster Descriptions <monsters_a-z/index>
```

Comment:
To be more specific, I _think_ that this essentially references the structure included in that index file. If I globbed them, it would build a new instance of that structure, which would make it appear twice in the toctree.